exp_events.json

```
@@ -1,4 +1,11 @@
 [
+    {
+        "type": "levelup",
+        "name": "Avallach",
+        "levels": 1,
+        "preserve_percentage": false,
```

```suggestion
        "preserve_percentage": true,
```
This should be `true`. It preserves the percentage of the level for your next level. So if you were 10% of the way through level 12, you would now be 10% of the way through level 13. Having it set to false means you will be exactly at level 13. I think there is a flag that can be set, or you can just manually change this json.

---

I just leveled up to 12 so I had 0 exp. Should we just always set it to true anyway?

---

We should set it to true anyways, just in case. It is possible some future change will adjust how leveling works and you might end up with additional retroactive xp on a previous level. In that case we want to make sure you keep your deserved xp. This is one of the advantages of the event sourced system. You should just be able to accept my suggested changes from the github web UI